.buildkite/aggregate.yml

```diff
@@ -0,0 +1,27 @@
+agents:
+  queue: "juliaecosystem"
+  sandbox.jl: true
+
+steps:
+  - label: "Aggregate"
+    env:
+      # For authentication with SSH deploy key
+      - DOCUMENTER_KEY: "U2FsdGVkX18NGGZPl0eBX2lEcjlS3N0fd5wOkzryp+bzVthWTE5ZCKKZ6gfUWhx7WtvK3aJ52v4tvpBl2kNOX9Nz0Ath11Ik321ziO+q38Gy74BqYOIlG9v54rOpzPolxh5Hgk9QFtKm95+noR55hx4T9Un2bse4hFURcN6aqurXTSeDvtFbmo6f9TnBdKKu2KeIyhP88nBPUzG+ngiOJQSFYuBMjr7mbd99OoR/q1mmZcNnLSJI19Lh0mNZ/I7pi4sfl3e3vI8ajbwrZFCWZ1bHsqARzQbj9D5y1TCUhTROIk0IP7pVhpz8orkgeKG1fH0vY+VJA4bW6Otbna4SQo5WjAplWDXZb8blNn+yZpvLQfi2lYEMp4wAJJHBxe3Aj8QCZDAqonheOB5/7fk0EtC+C9Q5I+krgsbO0XipIdi+K3gkKat07DxrQoH5CLJvitetjZuNGIsgyAznZBTfVXng5uoq8L3Wpl8RD11xOgiVGBuuzXvtKKsEYDVn7WcQj5B6vN6AR/b1OxCW3FEfZ63fgRTNqff/y6rS4wCLYDFYsTv1wx4azYvBAj/E6RuZ0nk1bMVaE4hUxKzePeBsRSyNVGhHr3/8TAxHB6ytpx1Mf9Tvk+Oz5BjDzpNv+8hqAqzU5bOyBLQBHHbXvBy6hdYQHPQIEpJojh11woNVMPJCmk4CI1SDxj0jkiMstgoFcfXN4qgnSgbUrtc3pYp80tVq8CcpI4H0SinYSeBZL+YxDUlCXhb4Tmct5g+lzBlRJoL5HnTvHM57Vsid1uomVQ=="
+
+    plugins:
+      - staticfloat/ssh-agent:
+          keyvars:
+            - "DOCUMENTER_KEY"
+      - staticfloat/cryptic#v2:
+          variables:
+            - DOCUMENTER_KEY="${DOCUMENTER_KEY?}"
```

Comment:
Sorry, you can just directly put the value here.  E.g.
```
- DOCUMENTER_KEY=....
```

Reply:
Will this have to be still specified as an `ENV`? I'd like to avoid having to explicitly specify the full encrypted value everywhere if I can avoid it.

Reply:
The way cryptic works is to take those values and turn them into environment variables.  So `cryptic` will export that value, and then `ssh-agent` can read it.

BTW I usually put `cryptic` before `ssh-agent` in the plugin ordering, to help this visually flow better.

Reply:
Ah, that makes sense. Thanks!

Reply:
Addressed in e2bfdab.